app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2804,7 +2804,6 @@ class BrowserTabFragment :
         popupMenu.dismiss()
         loginDetectionDialog?.dismiss()
         automaticFireproofDialog?.dismiss()
-        browserAutofill.removeJsInterface(webView)
```

Comment:
the `WebView` is being destroyed in the next line anyway, so there's no need to remove web message listeners at this point. 

Suspect trying to do so is resulting in a race condition in `WebView` between removing the listeners and destroying the `WebView`.